dashboard/db/migrate/20170524215951_add_pair_programming_enabled_to_section.rb

```diff
@@ -0,0 +1,5 @@
+class AddPairProgrammingEnabledToSection < ActiveRecord::Migration[5.0]
+  def change
+    add_column :sections, :pair_programming_enabled, :boolean, null: false, default: true
```

Comment:
Is `enabled` redundant? More succinct and equally descriptive (at least to me) would be `pairing_allowed` or `pair_programming`.